api/management/README.md

```diff
@@ -0,0 +1,200 @@
+              the old one, effectively replacing it.
+- **delete**: True deletion is not possible, in order to allow retrieval of the
+              specific policy version that was used to generate an attestation
+              result at a later date. However, all policies may be deactivated,
+              resulting in no policy being used during verification.
+
+[Attestation Results]: https://www.rfc-editor.org/rfc/internet-drafts/draft-fv-rats-ear-00.html
+[OPA Policy README]: https://github.com/veraison/services/blob/main/policy/README.opa.md#rules
+
+## REST API
+
+### Adding a new policy
+
+- A policy is added for a new scheme (with tenant implicit via auth).
+- Optionally, a name for the policy may be specified via query arguments. If a
+  name is not specified, the default name of `"default"` will be used.
+- On success, the new policy object is returned, with Location header set to
+  its unique URI.
+- The newly added policy is inactive. It must be activated via  a separate
+  request (see below).
+
+#### Request:
+```http
+POST /management/v1/policy/PSA_IOT&name=base HTTP/1.1
+Host: veraison.example
+Content-Type: application/vnd.veraison.policy.opa
+Accept: application/vnd.veraison.policy+json
+
+hardware = "AFFIRMING" {
+    // secured state
+    state := bits.rsh(bits.and(evidence["psa-security-lifecycle"], 0xFFFF0000), 8)
+    state >= 0x3000
+    state <= 0x30FF
+} else = "CONTRAINDICATED"
+```
+
+#### Response:
+```http
+HTTP/1.1 201 Created
+Content-Type: application/vnd.veraison.policy+json
+Location: /management/v1/policy/PSA_IOT/340d22f7-9eda-499f-9aa2-5af295d6d812
+
+{
+  "type": "opa",
+  "name": "base",
+  "uuid": "340d22f7-9eda-499f-9aa2-5af295d6d812",
+  "active": false,
+  "ctime": "2023-07-06 00:00:00",
+  "rules": "hardware = \"AFFIRMING\" {\n          // secured state\n          state := bits.rsh(bits.and(evidence[\"psa-security-lifecycle\"], 0xFFFF0000), 8)\n          state >= 0x3000\n          state <= 0x30FF\n      } else = \"CONTRAINDICATED\"\n  }"
+}
+```
+
+### Retrieving a specific policy object
+
+A policy may be retrieved via its unique URI.
+
+#### Request:
+```http
+GET /management/v1/policy/PSA_IOT/340d22f7-9eda-499f-9aa2-5af295d6d812 HTTP/1.1
+Host: veraison.example
+Accept: application/vnd.veraison.policy+json
+```
+
+#### Response:
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.veraison.policy+json
+
+{
+  "type": "opa",
+  "name": "base",
+  "uuid": "340d22f7-9eda-499f-9aa2-5af295d6d812",
+  "active": false,
+  "ctime": "2023-07-06 00:00:00",
+  "rules": "hardware = \"AFFIRMING\" {\n          // secured state\n          state := bits.rsh(bits.and(evidence[\"psa-security-lifecycle\"], 0xFFFF0000), 8)\n          state >= 0x3000\n          state <= 0x30FF\n      } else = \"CONTRAINDICATED\"\n  }"
+}
+```
+
+### Retrieving the active policy object
+
+The active policy object for a scheme may be retrieved by omitting the UUID.
```

Comment:
I think, it will add a value, if we state an alternative example flow where what return code is returned and if (200) then what Body is returned, in case there are no `active` policies in the store

Reply:
This only describes the general idea, this does not desribe error condtions (neither do  similar descriptions for the other API).  Doing so would clutter this high level explantion. 

The formal API description (the open API yaml document) covers that instead.

Reply:
@setrofim : A one liner stating whether an error is returned will help set the context more clearly.

Even in the components yaml file I am not clear what error code is returned for this specific case?

Reply:
I agree we shouldn't add clutter.  This is just a protocol overview that should only capture the intended happy-path flow.  The details are in the openapi docs.

Reply:
We do not define the specific errors -- this is the same for other API descriptions. The error is implicit in the fact   this is REST --  if you request a resource that does not exist, you get a 404. This a general convention, and is not part of this API per se.